pkg/rbac/userData.go

```diff
@@ -243,3 +243,15 @@ func (user *userData) getImpersonationClientSet(clientToken string, cache *Cache
 	}
 	return user.authzClient, nil
 }
+
+func (userdata *UserData) GetCsResources() []Resource {
+	return userdata.csResources
+}
+
+func (userdata *UserData) GetNsResources() map[string][]Resource {
+	return userdata.nsResources
+}
+
+func (userdata *UserData) GetManagedClusters() []string {
+	return userdata.managedClusters
+}
```

Comment:
Use `user` to stay consistent with the other methods.
```suggestion
func (user *UserData) GetCsResources() []Resource {
	return user.csResources
}

func (user *UserData) GetNsResources() map[string][]Resource {
	return user.nsResources
}

func (user *UserData) GetManagedClusters() []string {
	return user.managedClusters
}
```

Reply:
done